pkg/dashboard/apiserver/experiment/experiment.go

```diff
@@ -60,12 +60,14 @@ func NewService(
 	event core.EventStore,
 	config *config.ChaosDashboardConfig,
 	scheme *runtime.Scheme,
+	log logr.Logger,
 ) *Service {
 	return &Service{
 		archive: archive,
 		event:   event,
 		config:  config,
 		scheme:  scheme,
+		log: log,
```

Comment:
use `u.Log.WithName("experiments")` would be better

Reply:
This is a legacy method to log something, so this part is not a problem.